packages/react/src/common/options.ts

```diff
@@ -0,0 +1,39 @@
+import { FlagEvaluationOptions } from '@openfeature/web-sdk';
+
+export type ReactFlagEvaluationOptions = {
+  /**
+   * Suspend flag evaluations while the provider is not ready.
+   * Set to false if you don't want to show suspense fallbacks until the provider is initialized.
+   * Defaults to true.
+   */
+  suspendUntilReady?: boolean;
+  /**
+   * Suspend flag evaluations while the provider's context is being reconciled.
+   * Set to true if you want to show suspense fallbacks while flags are re-evaluated after context changes.
+   * Defaults to false.
+   */
+  suspendWhileReconciling?: boolean;
+  /**
+   * Update the component if the provider emits a ConfigurationChanged event.
+   * Set to false to prevent components from re-rendering when flag value changes
+   * are received by the associated provider.
+   * Defaults to true.
+   */
+  updateOnConfigurationChanged?: boolean;
+  /**
+   * Update the component when the OpenFeature context changes.
+   * Set to false to prevent components from re-rendering when attributes which
+   * may be factors in flag evaluation change.
+   * Defaults to true.
+   */
+  updateOnContextChanged?: boolean;
+} & FlagEvaluationOptions;
+
+export const DEFAULT_OPTIONS: ReactFlagEvaluationOptions = {
+  updateOnContextChanged: true,
+  updateOnConfigurationChanged: true,
+  suspendUntilReady: true,
+  suspendWhileReconciling: true,
+};
```

Comment:
I changed the defaults here! Now suspense it automatic for both reconciliation and readiness. I think this is the best choice, but I could also be convinced we could set both to false.

Either way, I think both suspense settings should match.

Reply:
I don't know much about suspense - I haven't personally worked on a codebase that uses it yet.

What effect would this have for codebases that haven't opted into Suspense at all?

Reply:
> What effect would this have for codebases that haven't opted into Suspense at all?

This would result in a runtime error @moredip. If you do not have a suspense boundary `<Suspense ...>` above your use of a suspense enabled data source it would simply let the "thrown" error trickle up.

Reply:
I think these are the right defaults @toddbaert.
But I think I would like an option like `enableSuspense` that overwrites the fine grained controls.
An application that does not use suspense could then just set this prop of the provider and would not have to set two options.

Reply:
Ya I can add that, as well as some typings that prevent nonsense configuration. Good idea.

Reply:
I did this.